### PR TITLE
updated docs for clientApplications

### PR DESCRIPTION
**Pull request by danielwood95**

### CODE REVIEW

File: api-reference/beta/resources/conditionalaccessconditionset.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ Represents the type of conditions that govern when the policy applies.
 | Property     | Type        | Description |
 |:-------------|:------------|:------------|
 |applications|[conditionalAccessApplications](conditionalaccessapplications.md)| Applications and user actions included in and excluded from the policy. Required. |
-|users|[conditionalAccessUsers](conditionalaccessusers.md)| Users, groups, and roles included in and excluded from the policy. Required. |
+|users|[conditionalAccessUsers](conditionalaccessusers.md)| Users, groups, and roles included in and excluded from the policy. Either **users** or **clientApplications** is required. |
+|clientApplications|[conditionalAccessClientApplications](../resources/conditionalaccessclientapplications.md)|Client applications (service principals, workload identities) included in and excluded from the policy. Either **users** or **clientApplications** is required. |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/conditionalaccessconditionset.md:24]:
```suggestion
|clientApplications|[conditionalAccessClientApplications](../resources/conditionalaccessclientapplications.md)|Client applications (service principals and workload identities) included in and excluded from the policy. Either **users** or **clientApplications** is required. |
```

File: changelog/Microsoft.IdentityProtectionServices.json [hunk 2]
```diff
@@ -1,5 +1,31 @@
 {
 	"changelog": [
+		{
+			"ChangeList": [
+				{
+					"Id": "cad83874-f5f9-459e-9b42-965603408cda",
+					"ApiChange": "Property",
+					"ChangedApiName": "clientApplications",
+					"ChangeType": "Addition",
+					"Description": "Added the **clientApplications** property to the [conditionalAccessConditionSet](https://docs.microsoft.com/en-us/graph/api/resources/conditionalAccessConditionSet?view=graph-rest-beta) resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityProtectionServices.json:10]: Suggestion to enhance the description
```suggestion
					"Description": "Added the **clientApplications** property to the [conditionalAccessConditionSet](https://docs.microsoft.com/en-us/graph/api/resources/conditionalAccessConditionSet?view=graph-rest-beta) resource. Use this property to configure the service principals and workload identities to include in or exclude from a conditional access policy.",
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @danielwood95 for raising this PR.
- Left suggestions for minor changes to resolve before approval
- Q: Should the scope of this change also include adding the **applicationFilter** property in **conditionalAccessApplications**?

The previous error was due to your branch being too out of sync with the **main** branch at the time you pushed the commit. Resolved it by merging in main and resolving the merge conflict in the changelog.

**Comment by danielwood95** [comment]
> Thank you @danielwood95 for raising this PR.
> 
> * Left suggestions for minor changes to resolve before approval
> * Q: Should the scope of this change also include adding the **applicationFilter** property in **conditionalAccessApplications**?
> 
> The previous error was due to your branch being too out of sync with the **main** branch at the time you pushed the commit. Resolved it by merging in main and resolving the merge conflict in the changelog.

Yes, this should include adding applicationFilter to conditionalAccessApplications. I can add that to this PR as well.

**Review by FaithOmbongi — APPROVED** [review]

**Pull request merged**